cli/functionary/login.py

```diff
@@ -21,10 +28,33 @@ def login_cmd(ctx, user, password, host):
     # check status code/message on return then exit
     if success:
         click.echo("Login successful!")
+        # save host for future commands on success
+        save_host(host)
+
     else:
         click.secho(
             message,
             err=True,
             fg="red",
         )
         ctx.exit(1)
+
+
+def save_host(host):
```

Comment:
This duplicates the similar call for the token.  Instead, we should refactor this so that there is a generic set of config handling functions.  Probably in a `config.py`, such that you can just do something like:

```
get_config("host")
set_config("host", "localhost:8000")
```

Reply:
Also, you'll want exception handling for situations such as the config file not being able to be read or written to.  In such cases, raise a `ClickException` because that will handle the output and exit code for you and save you from having to handle these exceptions in whatever code is calling your config handling functions.